camera_alignment_core/alignment_core.py

```diff
@@ -141,12 +141,11 @@ def align_image(
                 aligned_channel[z_index, :, :] = skimage.transform.warp(
                     unaligned_channel[z_index, :, :],
                     inverse_map=alignment_matrix,
-                    order=3,
+                    order=0, #3
```

Comment:
could we pass this as an argument instead of hard coding it?

Reply:
an arg with a default does seem more versatile, although I have no clue how often it would get used.

Reply:
change added in new commit